src/nodedef.cpp

```diff
@@ -654,22 +654,22 @@ void ContentFeatures::deSerialize(std::istream &is)
 
 		u8 tmp = readU8(is);
 		if (is.eof()) /* readU8 doesn't throw exceptions so we have to do this */
-			throw SerializationError("");
+			throw SerializationError("EOF");
```

Comment:
these are immediately caught without going anywhere so need to fill anything in btw

Reply:
Strange enough, using "" as a format string generates a warning:

warning: zero-length gnu_printf format string [-Wformat-zero-length]